.github/workflows/windows-ci.yaml

```diff
@@ -0,0 +1,147 @@
+name: Windows-CI
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: windows-latest
+
+    strategy:
+      matrix:
+        arch: ["x86", "x64"]
+        msvc: ["v141"]
+        qt: ["5.12.8"]
+
+    env:
+      CMAKE_GENERATOR: Visual Studio 16 2019
+      CMAKE_GENERATOR_TOOLSET: ${{ matrix.msvc }}
+      vcpkg-triplet: ${{ matrix.arch }}-windows-static-md
```

Comment:
I didn't know vcpkg shipped with this triplet now...  That's great news :)

Reply:
It's an "unsupported community triplet" - but it works well enough in my experience :smile: